### ISSUE TITLE
Custom XAML Analysis support for Visual Studio 2022

### ISSUE THREAD

**Issue by YakupIpek21** [mrlacey/Rapid-XAML-Toolkit#504]
**Is your feature request related to a problem? Please describe.**

No.

**Describe the solution you'd like**

To create custom analyzers for VS22.

**Comment by mrlacey** [comment]
VS2022 support will come in time.

**Comment by mrlacey** [comment]
X-Ref: #481

**Comment by chuongmep** [comment]
Please !

**Comment by mrlacey** [comment]
> Please !

Yes, I want to do this too but it's a lot of work and currently, I'm having to prioritize other (paid--or likely to lead to paid) work.
If you want to make a start yourself, I'm open to pull requests. Or sponsorship of the work.... ;)

**Comment by mrlacey** [comment]
Note to self.
See these breaking changes that will affect this: https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/breaking-api-list?view=vs-2022#legacy-find-api-deprecation

**Comment by shmuelie** [comment]
Looks like this is now supported in code but the VS marketplace entry has not been updated?

**Comment by mrlacey** [comment]
> Looks like this is now supported in code but the VS marketplace entry has not been updated?

new release coming "soon" - I merged all the WIP into the main branch just to get a handle on things and because it's close..
